requirements.txt

```diff
@@ -36,7 +36,7 @@ typing-extensions==3.7.4
 # A common package that holds the models deifnition and schemas that are used
 # accross different amundsen repositories.
 amundsen-common>=0.9.0
-amundsen-gremlin>=0.0.7
+amundsen-gremlin>=0.0.9
```

Comment:
could you update https://github.com/amundsen-io/amundsendatabuilder/blob/master/setup.py#L66 as well? the requirement.txt is only used in CI.

Reply:
Sure: https://github.com/amundsen-io/amundsendatabuilder/pull/473

Reply:
Opps or did you mean to move gremlin dependencies to be extras like: https://github.com/amundsen-io/amundsenmetadatalibrary/blob/master/setup.py#L26 ?

Reply:
nvm, I think you are correct, I somehow thought you are updating the databuilder.